docs/curation-review-tut.md

```diff
@@ -0,0 +1,570 @@
+# Cellxgene Curation Tutorial
+
+<!--- ![image](https://user-images.githubusercontent.com/25663501/111377133-b07a4c80-8676-11eb-8eb8-07ca4d7a77e9.png) --->
+
+<img align="right" src="https://user-images.githubusercontent.com/25663501/111377133-b07a4c80-8676-11eb-8eb8-07ca4d7a77e9.png">
+
+
+<!--- ## Overview --->
+<!--- purpose: Motivate submission to the data portal give context --->
+
+[Cellxgene's publishing platform](https://cellxgene.cziscience.com/) and [interactive single cell data explorer](https://github.com/chanzuckerberg/cellxgene) is a system which is optimized for access, exploration and reuse of single cell data. In order to achieve these goals, the cellxgene platform currently accepts curated [AnnData](https://anndata.readthedocs.io/en/latest/#) objects adhering to a [succinct data schema](corpora_schema.md). Adherence to a standardized data schema allows for efficient navigation and integration of the growing number of single cell datasets that are becoming available. In this tutorial, you will learn the essential information for curating a single cell dataset using CZI's curation tools and uploading to the data portal. Hosting your data on the cellxgene portal will offer the following benefits:
+ - link permanence (you can reference in your publication without ever worrying about dead links)
+ - sharing of private datasets with collaborators (keep the data private until it is ready for publication)
+ - no barrier for readers to explore your dataset (and no need for you to build your own single cell data explorer)
+ - accesibility of your dataset in the major single cell data formats (including `AnnData`, `seurat`, and `loom`)
+
+This tutorial will consist of an explanation of 1) how to create and structure an `AnnData` object with your single cell data 2) how to augment this object with information that is specific to the cellxgene schema and 3) how to upload this object to the cellxgene data portal. If you run into any issues during this tutorial, or have any suggestions on how to improve the portal and curation experience, you can contact us via [cellxgene@chanzuckerberg.com](cellxgene@chanzuckerberg.com).
```

Comment:
This tutorial explains how to:

* Create and structure an `AnnData` object with your single cell data
* Augment this object with information that is ~specific to~ required by the cellxgene schema
* Upload this ~~object~~ dataset to the cellxgene data portal
*

Reply:
I had something like this originally, but I wasn't wild about having three bulleted lists in quick succession